### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by MocapMike**
added pyqt5

### CODE REVIEW

File: requirements.txt
Comment (trentwirth) [inline requirements.txt:1]: looks great! Thanks!

File: requirements.txt [hunk 1]
```diff
@@ -7,4 +7,5 @@ pyopengl==3.1.6
 mediapipe==0.8.11
 keyboard==0.13.5
 scipy==1.9.1
```
Comment (trentwirth) [inline requirements.txt]:
```suggestion
scipy~=1.9.1
```
allows you to change the patch number

**Review by trentwirth — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (MocapMike) [reply requirements.txt:1]: Are you sure?!

**Review by trentwirth — APPROVED** [review]
ok looks good!

**Pull request merged**